xml/System.Net.WebSockets/ClientWebSocket.xml

```diff
@@ -90,7 +90,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>Aborts the connection and cancels any pending IO operations.</summary>
+        <summary>Aborts the connection and cancels any pending IO operations. The ClientWebSocket cannot be reused once aborted.</summary>
```

Comment:
As with @rpetrusha, seems like the addition would be better replacing the "To be added." below.